### PR TITLE
Fix TurnToAngle command

### PR DESCRIPTION
**Pull request by Jamdan2**
Use the left encoder to turn using pid.
Has been tested, seems to work with modifications. 
**_DO NOT MERGE YET_** (very important)

### CODE REVIEW

**Comment by andrewda** [comment]
Nobody resolve conflicts yet, please, no matter how tempting that button is.

File: build.gradle [hunk 1]
```diff
@@ -37,6 +37,7 @@ dependencies {
     compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlinVersion"
 
     compile "org.sert2521.sertain:core:$sertainVersion"
+    compile 'com.github.SouthEugeneRoboticsTeam:vision-plugin:ead1dfcdd4'
```
Comment (andrewda) [inline build.gradle]: Qué?

File: src/main/java/org/sert2521/powerup/drivetrain/Drivetrain.kt [hunk 2]
```diff
@@ -50,6 +51,7 @@ object Drivetrain : Subsystem() {
 
     override fun onTeleopStart() {
         rightJoystick.whenActive(12, SendToBottom() and IntakeBlock() and DriveToCube())
+        rightJoystick.whenActive(8, TurnToAngle(360.0))
```
Comment (andrewda) [inline src/main/java/org/sert2521/powerup/drivetrain/Drivetrain.kt]: Should remove before merging

File: build.gradle (hunk 1 above)
Comment (SUPERCILEX) [reply build.gradle]: Comes from a commit I rebased out of dev a few days ago. Aiden must have gotten it in that 30 second window, not bad! 😆

**Pull request merged**